app/src/main/java/org/dicekeys/app/MainActivity.kt

```diff
@@ -39,14 +45,10 @@ class MainActivity : AppCompatActivity() {
         binding.toolbar.setupWithNavController(navController, appBarConfiguration)
 
         navController.addOnDestinationChangedListener { _, destination, _ ->
-            // binding.appBarLayout.isInvisible = (destination.id == R.id.introFragment)
+            if(destination.id == R.id.listDiceKeysFragment){
```

Comment:
I can't figure out what this is doing.  Document?

Reply:
This is the easiest way to get the action navigating away from a DiceKeyFragment to remove the DiceKey from memory.
Going back with the hardware back button or from the toolbar back arrow are handled this way.